contrib/cookiecutter/ckan_extension/hooks/post_gen_project.py

```diff
@@ -36,15 +36,22 @@ def recut():
     setup_template = 'setup.py'
 
     # get context
-    context = {{ cookiecutter | jsonify }}
+    context = {}
+
+{%- for key, value in cookiecutter.items() -%}
+{% if value is not none %}
+    context["{{key}}"] = {{ value|tojson }}
```

Comment:
True and False as well as None inside something like a list or dict would lead to problems too. We'd want `repr` here but I don't think we get that in jinja2